TRB_label_assist/clustering.py

```diff
@@ -28,9 +29,26 @@
     'mean_shift'
 ]
 
+def CleanEntryTypeData(loc_df,loc_entry):
```

Comment:
nit: if this is a function, it should start with a lower case - only classes start with upper case in CamelCase

Reply:
fixed.